accord-core/src/main/java/accord/messages/Request.java

```diff
@@ -23,7 +23,16 @@
 
 public interface Request extends Message
 {
-    void process(Node on, Id from, ReplyContext replyContext);
     default long waitForEpoch() { return 0; }
     default long knownEpoch() { return waitForEpoch(); }
+    void preProcess(Node on, Id from, ReplyContext replyContext);
+    void process(Node on, Id from, ReplyContext replyContext);
+
+    /**
+     * Process the request without replying back
+     */
+    default void process(Node on)
+    {
+        throw new UnsupportedOperationException();
```

Comment:
I think this belongs in the `LocalRequest` interface not the `Request` interface.

Reply:
It's for replay on startup, when the request needs to be processed, but a response is no longer expected.

Reply:
But to avoid confusion, I moved it to `LocalRequest` for now, as suggested. Will put the required change for playback on startup in the next PR, where its purpose will be more clear. Cheers.